### PR TITLE
leaderboard hit API

### PR DESCRIPTION
**Pull request by christianbiring1**
- [ ] In this activity I had to continue working on the Leaderboard list app, sending and receiving data from/to the Leaderboard API.
> 

- [ ] Create a new game with the name of my choice by using the API.
- [ ] Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- [ ] Implement the form "Submit" button (sending data to the API).
- [ ] Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/add.js [hunk 1]
```diff
@@ -20,40 +17,74 @@ form.addEventListener('submit', (e) => {
...
+}
+
+async function getData(url) {
+  const a = await fetch(url);
+  const b = await a.json();
+  recentScore = b.result;
+  recentScore.forEach((score) => {
+    palmares(score);
+  });
+  localStorage.setItem('scores', JSON.stringify(recentScore));
```
Comment (mikethreels) [inline src/add.js]: - [x] please make use of ES6 arrow functions instead of the function keyword as per the project requirements.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Christian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] some functions which haven't been changed in this milestone are still using the function keyword, please make use of ES6 arrow functions instead of the function keyword as per the project requirements.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/add.js (hunk 1 above)
Comment (christianbiring1) [reply src/add.js]: Thanks, @mikethreels! I almost forget every time to use the arrow function :smiley:

**Review by chrystalme — APPROVED** [review]
# Hi @christianbiring1 , 
giphy

## Project Status ✅ Approve

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

## Optional Requirement

- [x] It would be nice to add validation for the `input fields`

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**